src/Support/ResolveInfoFieldsAndArguments.php

```diff
@@ -132,15 +143,67 @@ private function foldSelectionSet(SelectionSetNode $selectionSet, int $descend):
         return $fields;
     }
 
-    private function getValue(ArgumentNode $argumentNode)
+    /**
+     * @param ValueNode $value
+     * @return mixed
+     * @throws \Exception
+     */
+    private function getValue(ValueNode $value)
     {
-        $value = $argumentNode->value;
         if ($value instanceof VariableNode) {
             $variableName = $value->name->value;
 
             return $this->info->variableValues[$variableName] ?? null;
         }
+        // Scalar Types
+        if ($value instanceof IntValueNode || $value instanceof FloatValueNode || $value instanceof StringValueNode || $value instanceof BooleanValueNode || $value instanceof EnumValueNode) {
+            return $value->value;
+        }
+        // null Type
+        if ($value instanceof NullValueNode) {
+            return;
```

Comment:
Shouldn't this be more explicit and actually return `null`?

Reply:
Originally I returned null, but `styleci` changed to this way by default.